pkg/aws/helpers.go

```diff
@@ -535,18 +537,15 @@ func GetOperatorRolePolicyPrefixFromCluster(cluster *cmv1.Cluster, awsClient Cli
 		return rankedPolicyPrefix[0], nil
 	}
 
-	// If no commonly used prefix is found use operator role prefix
-	operatorRoleName, err := GetResourceIdFromARN(operatorRoles[0].RoleARN())
-	if err != nil {
-		return "", err
-	}
-	index := strings.LastIndex(operatorRoleName, "-openshift")
```

Comment:
Not sure if we should remove it, or stick to the current flow and have the LCP only as a fallback.

Reply:
This part is being done in backend now that's why I removed it and setup operator role prefix as last option fallback